README.md

```diff
@@ -32,6 +32,40 @@ Since the project is under active development, we recommend watching the reposit
 regularly for updates. Detailed documentation and usage instructions will be provided as the project
 progresses.
 
+## Testing the Mender MCU Client (POSIX)
+
+### Dependencies
+- cmake
+- libcurl4-openssl-dev
+- libcjson-dev
+- libmbedtls-dev
```

Comment:
I'd prefer these being generic things like `CMake`, `libcurl`, `cJSON` and `mbedTLS` and not Ubuntu/Debian-specific packages.

Reply:
I guess you can do both. List up what you need, and provide an example with an apt command for ubuntu